Source/GoBot/GoBot.swift

```diff
@@ -1575,13 +1575,16 @@ class GoBot: Bot {
     private var _statistics = BotStatistics()
 
     func statistics(queue: DispatchQueue, completion: @escaping StatisticsCompletion) {
+        let repoIdentity = self._statistics.repo.identity
```

Comment:
I think `_statistics` basically acts as an in-memory cache of the last known statistics. This means it will be nil the first time this function is called after the app starts, and it will have a copy of the previous identity if the user switched identities. The way it is written probably won't cause any bugs in the app today, but I think it would be simpler to say `let repoIdentity = identity` here because `identity` is the source of truth for the logged in user.
```suggestion
        let repoIdentity = identity
```